main/resource-access.bs

```diff
@@ -279,6 +279,25 @@ Issue:
 Pertaining to events and loss of control mitigation:
 https://github.com/solid/specification/issues/41#issuecomment-534679278
 
+### Representations ### {#representations}
+
+When a server creates a resource on HTTP `PUT`, `POST` or `PATCH` requests
+such that the request's representation data encodes an *RDF document*
+[[!RDF11-CONCEPTS]] (as determined by the `Content-Type` header), the server
+MUST respect `GET` requests on this resource when the value of the `Accept`
```

Comment:
Yes to the idea, but the phrasing of `respect` is a bit vague/unclear to me

Reply:
Updated to use "accept": https://github.com/solid/specification/pull/196/commits/4d8858654c4056ef53f0420cb9118f1f7ba2b9a7 ie. to honour, conform to, follow through client's wishes - matching 7231's "received, understood, and accepted" for 2xx. Will this do for now? Happy to revise later.